### ISSUE TITLE
New maintainers?

### ISSUE THREAD

**Issue by karlhorky** [pomber/code-surfer#118]
Hi @pomber! 👋

It seems like you don't have much time to devote to this project.

Would you be open to one or more new maintainers taking over? 

I don't have a lot of time myself, but I saw that @mhartington, @vlsi and @NieDzejkob have opened some issues and shown continued engagement with the project.

Would be really great to have some more movement on this project!

**Comment by pomber** [comment]
True, I've been focusing on Code Hike which may replace Code Surfer at some point in the future, so I'm not paying too much attention to the issues in this repo. 

I'm open to adding new maintainers if someone wants to take care of the project.

**Comment by karlhorky** [comment]
Ok posted on Twitter, let's see if we can get this thing started again: https://twitter.com/karlhorky/status/1394278922003652609

cc @trevordmiller

**Comment by vlsi** [comment]
I just watched Code Hike demo, and it is amazing!

By the way, Code Surfer is the best tool for code-heavy presentations for me even though I present once or twice a year only :)
